sass/_MissingTerm.scss

```diff
@@ -0,0 +1,21 @@
+.CoveoMissingTerm {
+  margin: 0;
+  font-size: 12px;
+  padding: 0;
+  .coveo-field-caption {
+    margin-right: 5px;
+    color: #1e253f;
+    opacity: 0.5;
+  }
+  .coveo-missing-term {
+    color: #004990;
```

Comment:
You can include the @clickable() mixin here, and remove most of those rules.

Check other files to see how you can import the mixin and use it.

Reply:
Thanks, I added it to the `coveo-missing-term` since it is the only one clickable